Examples/Examples/BookmarksExampleView.swift

```diff
@@ -31,44 +29,38 @@ struct BookmarksExampleView: View {
-                    Button {
-                        showingBookmarks.toggle()
-                    } label: {
-                        Label(
-                            "Show Bookmarks",
-                            systemImage: "bookmark"
-                        )
-                    }
-                    .popover(isPresented: $showingBookmarks) {
-                        // Display the `Bookmarks` components with a pre-defined
-                        // list of bookmarks. Passing in a `Viewpoint` binding
-                        // will allow the `Bookmarks` component to handle
-                        // bookmark selection.
-                        Bookmarks(
-                            isPresented: $showingBookmarks,
-                            mapOrScene: dataModel.map,
-                            viewpoint: $viewpoint
-                        )
-                        
-                        // Display the `Bookmarks` component with the list of
-                        // bookmarks in a map.
-//                        Bookmarks(
-//                            isPresented: $showingBookmarks,
-//                            mapOrScene: map
-//                        )
-                        // In order to handle bookmark selection changes
-                        // manually, use `onSelectionChanged(perform:)`.
-//                        .onSelectionChanged {
-//                            viewpoint = $0.viewpoint
-//                        }
+        MapViewReader { mapViewProxy in
+            MapView(map: map, viewpoint: viewpoint)
+                .onViewpointChanged(kind: .centerAndScale) {
+                    viewpoint = $0
+                }
+                .toolbar {
+                    ToolbarItem(placement: .primaryAction) {
+                        Button {
+                            showingBookmarks.toggle()
+                        } label: {
+                            Label(
+                                "Show Bookmarks",
+                                systemImage: "bookmark"
+                            )
+                        }
+                        .popover(isPresented: $showingBookmarks) {
+                            // Display the `Bookmarks` component with the list of
+                            // bookmarks in a map.
+                            Bookmarks(
+                                isPresented: $showingBookmarks,
+                                mapOrScene: map
+                            )
+                            // In order to handle bookmark selection changes
+                            // manually, use `onSelectionChanged(perform:)`.
+                            .onSelectionChanged { bookmark in
+                                if let viewpoint = bookmark.viewpoint {
+                                    Task { await mapViewProxy.setViewpoint(viewpoint, duration: 1) }
+                                }
+                            }
```

Comment:
Since we support viewpoint animation now, we can uncomment this and make good use of it.